tests/Yaapii.Atoms.Tests/IO/LoggingOutputTest.cs

```
@@ -0,0 +1,167 @@
+
+                var inputStream = new InputOf(new Uri(tempfile.Value())).Stream();
+                var content = "";
+                using (var reader = new StreamReader(inputStream))
+                {
+                    content = reader.ReadToEnd();
+                }
+
+                Assert.Equal(
+                    bytes,
+                    new BytesOf(content).AsBytes()
+                );
+            }
+        }
+        
+        [Fact]
+        public void LogWriteToLargeTextFile()
+        {
+            using (var tempfile = new TempFile("txt"))
+            {
+
+                try {
+                    var append = new AppendTo(tempfile.Value());
+
+                    var output =
+                        new LoggingOutput(
+                            append,
+                            "text file"
+                        ).Stream();
+
+                    var length =
+                        new LengthOf(
+                            new TeeInput(
+                                new ResourceOf("Assets/Txt/large-text.txt", this.GetType()),
+                                new OutputTo(output)
+                            )
+                        ).Value();
+
+                    append.Dispose();
+
+                    var inputStream = new InputOf(new Uri(tempfile.Value())).Stream();
+                    var content = "";
+                    var input = "";
+                    using (var reader = new StreamReader(inputStream))
+                    {
+                        content = reader.ReadToEnd();
+                    }
+                    using (var reader = new StreamReader(new ResourceOf("Assets/Txt/large-text.txt", this.GetType()).Stream()))
+                    {
+                        input = reader.ReadToEnd();
+                    }
+
+                    Assert.Equal(
+                        input,
+                        content
+                    );
+
+
+                }catch(Exception ex)
```

Please do not catch Exceptions in tests because we do not want to hide errors in tests.